src/components/ProgressCircle.ts

```diff
@@ -116,6 +121,15 @@ export class ProgressCircle extends Component<ProgressCircleProps, { alertMessag
                 progressText = `${value}`;
             } else if (text === "percentage") {
                 progressText = progress + "%";
+            } else if (text === "static" && this.props.displayTextStatic) {
+                progressText = !(this.props.displayTextStatic.includes("{value}"))
+                    ? `${progress} ${this.props.displayTextStatic}`
+                    : this.props.displayTextStatic
+                        .replace("{value}", String(value))
+                        .replace("{maximumValue}", String(this.props.maximumValue));
```

Comment:
Never use `String` with capital case without `new` check difference between String and string
https://stackoverflow.com/questions/17256182/what-is-the-difference-between-string-literals-and-string-objects-in-javascript

Should use `.toString()` anyway... https://mikemcl.github.io/big.js/#toS

Reply:
Changed